src/components/Dashboard.tsx

```diff
@@ -391,13 +414,13 @@ function Dashboard() {
   const [columns, setColumns] = useState<Array<ColomnType>>(baseColumns)
   const [summary, setSummary] = useState<Summary>(deepClone(emptySummary))
   const [wallets, setWallets] = useState<Array<Wallet>>([])
-  const [paywall, setPaywall] = useState<any>()
```

Comment:
Paywall moved outside of the component lifecycle so that it does not reload when state changes... etc